cg/meta/orders/schema.py

```diff
@@ -110,22 +110,22 @@ def validate(self, value):
 
     # This information is required
     'name': validators.RegexValidator(NAME_PATTERN),
-    'container': validators.Any(CONTAINER_OPTIONS),
+    'container': validators.Optional(TypeValidator(str, allow_none=True), None),
```

Comment:
Should prolly move to the "Order portal specific" block on line 109

Reply:
The meaning of "Order portal specific" in this context is whether the attribute exists in the Orderform or not and thus I think it is placed under the correct comment? 

What might be a bit confusing here is that some fields are required when the information comes from the Orderform but is optional when you select an already existing sample in the Orderportal.  Maybe we should implement something along the lines of OrValidator( 'internal_id', 'container' ) ?